src/Avalonia.Native/WindowImplBase.cs

```diff
@@ -70,9 +70,9 @@ public void Resize(Size clientSize, WindowResizeReason reason)
             Native?.Resize(clientSize.Width, clientSize.Height, (AvnPlatformResizeReason)reason);
         }
         
-        public override void SetFrameThemeVariant(PlatformThemeVariant themeVariant)
+        public override void SetFrameThemeVariant(PlatformThemeVariant? themeVariant)
         {
-            Native?.SetFrameThemeVariant((AvnPlatformThemeVariant)themeVariant);
+            Native?.SetFrameThemeVariant((AvnPlatformThemeVariant)(themeVariant ?? PlatformThemeVariant.Light));
```

Comment:
Probably should use PlatformSettings.GetCurrentColors() when themeVariant is null.

Reply:
Done